conf/application.conf

```diff
@@ -80,18 +83,22 @@ microservice {
   }
 }
 
+mongodb {
+  uri = "mongodb://localhost:27017/customs-exports-internal"
+}
+
 metrics {
   name = ${appName}
   rateUnit = SECONDS
   durationUnit = SECONDS
-  showSamples = true
-  jvm = true
-  enabled = true
+  showSamples = false
+  jvm = false
+  enabled = false
 }
 
 auditing {
-  enabled = true
-  traceRequests = true
+  enabled = false
```

Comment:
We could do with noting somewhere that we need to override these in  app-config-base

Reply:
Your propose is to change it to previous implementation? I mean enable metrics and auditing locally?

Reply:
as far as i'm aware, auditing is the only one we need to turn off locally. I'd look at the config for other frontends